thespeechmachine/install.bash

```diff
@@ -3,7 +3,7 @@
 # Install vbox if not installed
 if ! dpkg -s virtualbox-4.3 &> /dev/null
 then
-    URL='http://download.virtualbox.org/virtualbox/4.3.20/virtualbox-4.3_4.3.20-96996~Ubuntu~precise_amd64.deb'; FILE=`mktemp`; wget "$URL" -qO $FILE && sudo dpkg -i $FILE; rm $FILE
+    URL='http://download.virtualbox.org/virtualbox/4.3.20/virtualbox-4.3_4.3.20-96996~Ubuntu~precise_amd64.deb'; FILE=$(mktemp); wget "$URL" -qO "$FILE" && sudo dpkg -i "$FILE"; rm "$FILE"
```

Comment:
Remove semi-colons and split into multiple lines

Reply:
done